Add runtime logs regex search docs  and add system logs in PE docs (#1874)
wso2/docs-choreo-dev: en/developer-docs/docs/monitoring-and-insights/view-logs.md

    ![Runtime logs](../assets/img/monitoring-and-insights/view-logs/runtime-logs.png)

> ### Searching on runtime logs

Reviewer: ```suggestion
### Searching through runtime logs
```

Author: Done through https://github.com/wso2/docs-choreo-dev/pull/1874/commits/fcb36dcf8b5428ffccde872b698f50730df8db5d

The runtime logs interface provides powerful search capabilities to help you quickly locate specific log entries.

#### Text search

> Enter any text in the search box to find log entries containing that exact phrase. Searches are case-sensitive and the search will match partial strings within log messages of Application, Gateway and System logs.

Reviewer: ```suggestion
Type any text into the search box to locate log entries that contain that exact phrase. Searches are case-sensitive and will match partial strings within the log messages of Application, Gateway, and System logs.
```

Author: Done through https://github.com/wso2/docs-choreo-dev/pull/1874/commits/fcb36dcf8b5428ffccde872b698f50730df8db5d

#### Advanced search with regex

> Use Lucene compatible regex patterns for complex search queries. Refer to the [OpenSearch regex syntax](https://docs.opensearch.org/latest/query-dsl/regex-syntax/#standard-regex-operators) for more details.

Reviewer: ```suggestion
Use Lucene-compatible regex patterns to perform advanced search queries. Refer to the [OpenSearch regex syntax](https://docs.opensearch.org/latest/query-dsl/regex-syntax/#standard-regex-operators) for more details.
```

Author: Done through https://github.com/wso2/docs-choreo-dev/pull/1874/commits/fcb36dcf8b5428ffccde872b698f50730df8db5d

Pattern examples:

  - `error.*timeout` : Find logs containing "error" followed by "timeout"

>   - `.*(GET|POST).*&.*500.*` : Find logs of GET or POST requests with HTTP 500 status code

Reviewer: ```suggestion
  - `.*(GET|POST).*&.*500.*` : Find logs of GET or POST request methods with HTTP 500 status code
```

Author: Done through https://github.com/wso2/docs-choreo-dev/pull/1874/commits/fcb36dcf8b5428ffccde872b698f50730df8db5d
